README.md

```diff
@@ -121,6 +121,9 @@ Optional arguments:
     It is enforced on per-second basis, to avoid bursts.
 - `--traveltime-max-rpm [int]`: Set max number of parallel requests sent to TravelTime API per minute. Default is 60.
   It is enforced on per-second basis, to avoid bursts.
+- `--providers [providers]`: The providers you want to compare to TravelTime (e.g., --providers google mapbox).
```

Comment:
maybe it would be easier to use config file instead of args?

Reply:
Plus if we don't select all providers, apiKeys should be optional for missing ones.

Reply:
I can move it to a config file I suppose, but then we'd have 2 places for config.

And apiKeys are optional for those that aren't selected.

Reply:
Maybe it's a good excuse to move any provider related config to a file, including max_rpm args, etc, so we don't have so many args. The same could be done with api keys, instead of env variables

Reply:
yeap cause we have more and more params, so it would be easier to set them up in a config